R-Python-Testing/write_python_read_r.py

```diff
@@ -0,0 +1,75 @@
+import shutil
```

Comment:
when you consider that we will need a bunch of these tests, how will the individual tests be organized and dispatched?

One possibility:  just make all of this a set of stand-alone pytest tests, and use that framework for the very top-level dispatching, logging and error handling?

Reply:
My original idea was to make this a top-level module and create several pytest files (one for each data type).